tests/app/template/test_rest.py

```diff
@@ -534,17 +502,15 @@ def test_update_should_update_a_template(client, sample_user, sample_service, sa
     assert update_response.status_code == 200
     update_json_resp = update_response.get_json()
     assert update_json_resp['data']['content'] == new_content
-    assert update_json_resp['data']['postage'] == 'first'
     assert update_json_resp['data']['name'] == template.name
     assert update_json_resp['data']['template_type'] == template.template_type
     assert update_json_resp['data']['version'] == 2
 
 
-@pytest.mark.xfail(reason='Mislabelled for route removal, fails when unskipped', run=False)
 def test_should_be_able_to_archive_template(notify_db_session, client, sample_template):
     template = sample_template()
     data = {
-        'name': template.name,
+        'name': 'test template',
```

Comment:
Don't hard code things like this to avoid concurrent test failures.  Use `str(uuid4())`.

Reply:
Updated.